File: net/Link.Message.Client/content/VideoMessageContent.cs
namespace Link.Message.Client.content
{
	/// <summary>
	/// 视频消息内容
	/// </summary>
	public class VideoMessageContent : FileMessageContent
	{

		public VideoMessageContent(string downloadUrl, int size, string fileName, string extension) : base(downloadUrl, size, fileName, extension)
		{
		}

		public VideoMessageContent(string downloadUrl, int size, string fullFileName) : base(downloadUrl, size, fullFileName)
		{
		}
	}

}

File: java/link-message-client/src/main/java/link/message/client/content/VideoMessageContent.java
package link.message.client.content;

/**
 * 视频消息内容
 * @author zhongt
 *
 */
public class VideoMessageContent extends FileMessageContent {
	
	public VideoMessageContent(String downloadUrl, int size, String fileName, String extension) {
		super(downloadUrl, size, fileName, extension);
	}
	
	public VideoMessageContent(String downloadUrl, int size, String fullFileName) {
		super(downloadUrl, size, fullFileName);
	}
}
